### DISCUSSION TITLE
How should 'excluded-libraries' work?

### DISCUSSION THREAD

**Discussion by myrison** [Taxel/PlexTraktSync#874] [Support] [answered]
Thank you for this software!

I've added a number of private libraries to my 'excluded-libraries' section in config.json, but still see the logs complaining about items from within playing because it can't find a metadata provider.  Have I configured this correctly or should I submit an issue because it's not working as intended?

config.json
```
    "excluded-libraries": [
        "Private",
        "Family Holidays",
        "Fitness Videos",
        "Home Videos,"
        "Audiobooks (new)"
    ]
```

And here are the logs as one of the items is playing from my audiobooks library:

> 2022-04-04 19:14:13,975 ERROR[PlexTraktSync]:<Track:252703:He-Who-Fights-with-M>: Unable to parse a valid provider from guid com.plexapp.agents.audnexus://1039400205/-1/9?lang=en
> 2022-04-04 19:14:13,975 ERROR[PlexTraktSync.WatchStateUpdater]:on_play: Not found: /library/metadata/252703

I'm assuming that if something is in an excluded library, it shouldn't post these complaints in the logs, but maybe it does that regardless.   Thanks!

**Comment by simonc56** [comment, marked as answer]
The "excluded-libraries" is not implemented in the `watch` command. Only for the sync.

As **glensc** [said](https://github.com/Taxel/PlexTraktSync/issues/860#issuecomment-1085893911) :
> also, this project used to have only "sync" command, the "watch" was added as an experiment and seems it provided to be quite a useful one.

**Reply by myrison** [reply]
Thank you for the quick answer!  Glad to hear it's working as intended.

**Reply by glensc** [reply]
It also means having filtering in watch command should be submitted as feature request issue.
